src/RecoveryServices/RecoveryServices.Backup.Autorest/custom/Cmdlets/Policy/Edit-AzRecoveryServicesBackupSchedulePolicyClientObject.ps1

```diff
@@ -0,0 +1,387 @@
+﻿function Edit-AzRecoveryServicesBackupSchedulePolicyClientObject {
+	[OutputType('Microsoft.Azure.PowerShell.Cmdlets.RecoveryServices.Models.Api20230201.IProtectionPolicy')]
+    [CmdletBinding(PositionalBinding=$false)]
+    # RsvRef: should we call it workload type
+    [Microsoft.Azure.PowerShell.Cmdlets.RecoveryServices.Description('Edits the created default policy object.')]
+
+	param (
+        [Parameter(ParameterSetName='ModifySchedulePolicy', Mandatory, HelpMessage='Specifies the policy to be edited.')]
+        [Microsoft.Azure.PowerShell.Cmdlets.RecoveryServices.Models.Api20230201.IProtectionPolicy]
+        ${Policy},
+        
+        [Parameter(ParameterSetName='ModifySchedulePolicy', Mandatory, HelpMessage='Specifies the data source type.')]
+        [Microsoft.Azure.PowerShell.Cmdlets.RecoveryServices.Support.DatasourceTypes]
+        ${DatasourceType},
+
+        # AzureVM specific parameters
+
+        [Parameter(ParameterSetName='ModifySchedulePolicy', HelpMessage='Specifies the policy sub type for AzureVM.')]
+        [ValidateSet("Standard", "Enhanced", ErrorMessage = "Invalid value for PolicySubType. Please provide a valid policy sub type. Valid values are 'Standard' and 'Enhanced'.")]
+        [AllowEmptyString()]
+        [string]
+        ${PolicySubType} = $null,
```

Comment:
By default we can keep PolicySubType as standard instead of $null(even if user doesn't pass anything) for other workloads too.

Reply:
again we can't say undefined. we can keep standard itself as the default value